versioned_docs/version-v0.53/api/bridge/index.md

```diff
@@ -8,21 +8,31 @@ import EthAddresses from '@site/src/components/EthAddresses';
 
 There are three main components to the Vega <-> Ethereum bridge
 
-## [Staking bridge](./interfaces/IStake.md)
+## [Staking Bridge](./interfaces/IStake.md)
 
 <EthAddresses frontMatter={frontMatter} show={["StakingBridge"]} />
 
 Allows users to stake locked or unlocked Vega tokens.
 
 For an introduction to staking on Vega, [check out our Concepts section](../../concepts/vega-chain.md#bridges-used-for-staking) or for a higher level overview [see this blog post](https://blog.vega.xyz/staking-on-vega-17f22113e3df).
 
-## [ERC20 asset bridge](./interfaces/IERC20_Bridge_Logic.md)
+## [ERC20 Asset Bridge](./interfaces/IERC20_Bridge_Logic.md)
 <EthAddresses frontMatter={frontMatter} show={["ERC20Bridge"]} />
 
 Used for depositing ERC20 tokens in to a Vega network.
 
 
 To read more about how the bridge works, [see this blog post](https://blog.vega.xyz/vega-erc20-bridge-331a5235efa2). If you're looking for a way to deposit tokens from Sepolia on to the Fairground testnet, head over to [Fairground Console](https://console.fairground.wtf).
 
+## ERC20 Asset Pool
+<EthAddresses frontMatter={frontMatter} show={["ERC20AssetPool"]} />
+
+This is where deposited assets are stored. It is isolated from the Asset Bridge to minimize the chance of bugs in the more complex logic code and allows for painless future upgrades.
+
+To read more about how the bridge works, [see this blog post](https://blog.vega.xyz/vega-erc20-bridge-331a5235efa2). If you're looking for a way to deposit tokens from Sepolia on to the Fairground testnet, head over to [Fairground Console](https://console.fairground.wtf).
```

Comment:
```suggestion
To read more about how the bridge works, [see this blog post](https://blog.vega.xyz/vega-erc20-bridge-331a5235efa2). 
```

I've cut this Sepolia/Console sentence as it's the mainnet version and I don't want to confuse people.